docs/src/main/sphinx/connector/exasol.rst

```diff
@@ -0,0 +1,320 @@
+  * - ``REAL``
+    - ``DOUBLE PRECISION``
+    -
+  * - ``DOUBLE``
+    - ``DOUBLE PRECISION``
+    -
+  * - ``VARCHAR``
+    - ``VARCHAR(2000000)``
+    -
+  * - ``VARCHAR(n)``
+    - ``VARCHAR(n)``
+    - See :ref:`exasol character mapping`
+  * - ``CHAR(n)``
+    - ``CHAR(n)``
+    - See :ref:`exasol character mapping`
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``TIMESTAMP``
+    - ``TIMESTAMP``
+    -
+
+No other types are supported.
+
+.. _exasol number mapping:
+
+Mapping numeric types
+^^^^^^^^^^^^^^^^^^^^^
+
+An Exasol ``DECIMAL(p, s)`` maps to Trino's ``DECIMAL(p, s)`` and vice versa
+except in these conditions:
+
+- No precision is specified for the column (example: ``DECIMAL`` or
+  ``DECIMAL(*)``).
+- Scale (``s`` ) is greater than precision.
+- Precision (``p`` ) is greater than 36.
+- Scale is negative.
+
+.. _exasol character mapping:
+
+Mapping character types
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Trino's ``VARCHAR(n)`` maps to ``VARCHAR(n)`` and vice versa if ``n`` is no greater
+than 2000000. Exasol does not support longer values.
+If no length is specified, the connector uses 2000000.
+
+Trino's ``CHAR(n)`` maps to ``CHAR(n)`` and vice versa if ``n`` is no greater than 2000.
+Exasol does not support longer values.
+
+.. include:: jdbc-type-mapping.fragment
+
+.. _exasol-sql-support:
+
+SQL support
+-----------
+
+The connector provides read access and write access to data and metadata in
+Exasol. In addition to the :ref:`globally available <sql-globally-available>`
+and :ref:`read operation <sql-read-operations>` statements, the connector
+supports the following statements:
+
+* :doc:`/sql/delete`
+* :doc:`/sql/truncate`
+* :doc:`/sql/create-table`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/alter-table`
+* :doc:`/sql/comment`
+
+SQL INSERT
+^^^^^^^^^^
+
+Please note that ``INSERT`` and ``CREATE TABLE AS`` is not supported because
+this would require `parallel connections with the Exasol JDBC driver
+<https://exasol.my.site.com/s/article/Parallel-connections-with-JDBC?language=en_US>`_
+which cannot be implemented with the current architecture of Trino. Limiting the
+worker count by setting ``task.writer-count`` to 1 in Trino is also not a viable option as discussed in
+`issue #16011 <https://github.com/trinodb/trino/issues/16011>`_.
+
+
+SQL DELETE
+^^^^^^^^^^
+
+If a ``WHERE`` clause is specified, the ``DELETE`` operation only works if the
+predicate in the condition clause can be fully pushed down to the data source.
+
+Exasol does not support predicates ``<``, ``>`` and ``!=`` for the condition of ``DELETE`` statements.
+The statement will fail with a database error in these cases.
+
+.. include:: alter-table-limitation.fragment
+
+Table functions
+---------------
+
+The connector provides specific :doc:`table functions </functions/table>` to
+access Exasol.
+
+.. _exasol-query-function:
+
+``query(varchar) -> table``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``query`` function allows you to query the underlying database directly. It
```

Comment:
This whole wording and section might have to be updated ... there are some changes around this that recently got merged.

Reply:
I am not sure how to change this. I used the Oracle documentation as an example and that contains a similar description.